Add advantages, description, and hardware related documentation (#41)
PhotonVision/photonvision-docs: source/docs/getting-started/about/advantages.rst

Advantages
==========
PhotonVision has a myriad of advantages over similar solutions, including: 

Easy to Use User Interface
^^^^^^^^^^^^^^^^^^^^^^^^^^
The PhotonVision user interface is simple and modular, making things easier for the user. With a simpler interface, you can focus on what matters most, tracking targets, rather than how to use our UI. A major unique quality is that PhotonVision includes an offline copy of our documentation for your ease of access at competitions.

> User Calibratibration

Reviewer: ```suggestion
User Calibration
```
Fix typo

^^^^^^^^^^^^^^^^
Using PhotonVision allows the user to calibrate for their specific camera, which will get you the best tracking results. This is extremely important as every camera (even if it is the same model) will have it's own quirks and user calibration allows for those to be accounted for.

High FPS Processing
^^^^^^^^^^^^^^^^^^^

> Compared to the alternative softwares, PhotonVision boasts higher frames per second which allows for a smoother video stream and detection of targets. This ensures that you get the greatest accuracy possible at time of your shot.

Reviewer: ```suggestion
Compared to alternative software, PhotonVision boasts higher frames per second which allows for a smoother video stream and detection of targets. This ensures that you get the greatest accuracy possible at time of your shot.
```

> Compared to the alternative softwares, PhotonVision boasts higher frames per second which allows for a smoother video stream and detection of targets. This ensures that your get the greatest accuracy possible at time of your shot.

Reviewer: ```suggestion
Compared to the alternative softwares, PhotonVision boasts higher frames per second which allows for a smoother video stream and detection of targets. This ensures that your get the greatest accuracy possible at time of you shot.
```
Fix typo

Fully Open Source
^^^^^^^^^^^^^^^^^

> You can find all of our code on `GitHub <https://github.com/PhotonVision>`_, including code for our main program, documentation, vendor depenency (PhotonLib), and more. This helps you see everything working behind the scenes and increases transparency. This also allows users to make pull requests for features that they want to add in to PhotonVision that will be reviewed by the PhotonVision team. PhotonVision is liscened under the GNU General Public License (GPLv3) which you can learn more about `here <https://www.gnu.org/licenses/quick-guide-gplv3.html>`_.

Reviewer: ```suggestion
You can find all of our code on `GitHub <https://github.com/PhotonVision>`_, including code for our main program, documentation, vendor dependency (PhotonLib), and more. This helps you see everything working behind the scenes and increases transparency. This also allows users to make pull requests for features that they want to add in to PhotonVision that will be reviewed by the PhotonVision team. PhotonVision is licensed under the GNU General Public License (GPLv3) which you can learn more about `here <https://www.gnu.org/licenses/quick-guide-gplv3.html>`_.
```

Fix 2 typos
